assets/js/components/ChecksResults/CheckResult.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+import 'intersection-observer';
```

Comment:
Is this `intersection-observer`used for something in the test?

Reply:
It wasn't :+1: